lib/components/ClinicUserEdit.js

```diff
@@ -68,8 +68,10 @@ var MONTHS = [
 
 var ClinicUserEdit = React.createClass({
   propTypes: {
-    errorMessage: React.PropTypes.string,
-    errorDismissed: React.PropTypes.bool.isRequired,
+    createErrorMessage: React.PropTypes.string,
```

Comment:
to keep the naming consistent maybe these should be `createProfileErrorMessage` if the other is `updateProfileErrorMessage`. Thoughts?